src/http/common/entities/operator.ts

```diff
@@ -18,4 +24,28 @@ export class Operator {
     description: 'This value shows if node operator active',
   })
   active: boolean;
+
+  @ApiProperty({ description: 'Operator name' })
+  name: string;
+
+  @ApiProperty({ description: 'Ethereum 1 address which receives stETH rewards for this operator' })
+  rewardAddress: string;
+
+  @ApiProperty({ description: 'The maximum number of validators to stake for this operator' })
+  stakingLimit: number;
```

Comment:
I would say that this is the number of keys vetted by the DAO and that can be used for the deposit. So that there is no confusion with the target limit